Telerik.JustMock/Expectations/Abstraction/INonPublicExpectation.cs

```diff
@@ -72,6 +72,32 @@ public interface INonPublicExpectation
 		/// <returns>Reference to setup actions calls</returns>
 		FuncExpectation<TReturn> Arrange<TReturn>(object target, MethodInfo method, params object[] args);
 
+		/// <summary>
+		/// Arranges a property for mocking.
+		/// </summary>
+		/// <param name="target">Target instance</param>
+		/// <param name="propertyName">Property name</param>
+		/// <param name="value">Property value</param>
+		/// <returns>Reference to setup actions calls</returns>
+		ActionExpectation ArrangeSet(object target, string propertyName, object value);
```

Comment:
This summary can be improved. Please check what is the summary in for Mock.ArrangeSet.